app/models/setting.rb

```diff
@@ -67,6 +67,15 @@ class Setting < RailsSettings::Base
       secret: ENV['GITLAB_SECRET'],
     }, validates: { json: { format: :hash } }
 
+    field :github, type: :hash, display: true, restart_required: true, default: {
+      enabled: ActiveModel::Type::Boolean.new.cast(ENV['GITHUB_ENABLED'] || false),
+      site: ENV['GITHUB_SITE'] || 'https://github.com/login/oauth/authorize',
+      scope: ENV['GITHUB_SCOPE'] || 'user,repo,gist',
+      app_id: ENV['GITHUB_CLIENT_ID'],
+      secret: ENV['GITHUB_CLIENT_SECRET'],
+      required_org: ENV['GITHUB_REQUIRED_ORG'] || nil
```

Comment:
```suggestion
      required_org: ENV['GITHUB_REQUIRED_ORG']
```

alway returns `nil` if not fetch or not exists